.ebextensions/nginx/conf.d/move.mil.conf

```diff
@@ -0,0 +1,46 @@
+upstream move.mil-rails {
```

Comment:
To the best of my understanding, the `name` of the upstream only matters insofar as its referenced in the `proxy_pass http://move.mil-rails;` line further down in this file. The name of the Unix socket (`my_app.sock`) is set by the Puma configuration which (at the moment) is automatically generated. Hoping we can name that something more sensible down the line.